src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2419,7 +2419,6 @@ internal void Run(bool inputLoopIsNested)
                 Dbg.Assert(ui != null, "Host.UI should return an instance.");
```

Comment:
@vexx32, your last commit had 89 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Online help tests for PowerShell Cmdlets.Error occurred in Describe block
```powershell
Cannot validate argument on parameter 'Name'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
At line:16 char:34
```
ConsoleHost unit tests.ShellInterop.Verify Validate Dollar Error Populated should throw exception
```powershell
Expected an exception, with FullyQualifiedErrorId 'CommandNotFoundException' to be thrown, but the FullyQualifiedErrorId was 'ParameterArgumentValidationError,Microsoft.PowerShell.Commands.GetCommandCommand'. from D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1:108 char:22
    + ?        $a | & $powershell -noprofile -command { wgwg-wrwrhqwrhrh35h3h ?
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 109
109:             } | Should -Throw -ErrorId "CommandNotFoundException" -PassThru
```
ConsoleHost unit tests.-Login pwsh switch.Error occurred in Context block
```powershell
Cannot validate argument on parameter 'Name'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
At line:16 char:34
```
ConsoleHost unit tests.Pipe to/from powershell.errors are in text if error is redirected, encoded command, non-interactive, and outputformat specified
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 416
416:             $out | Should -Not -BeNullOrEmpty
```
Additional tests for Import-Module with WinCompat.Tests that ErrorAction/WarningAction have effect when Import-Module with WinCompat is used.Verify that Error is generated with default ErrorAction
```powershell
Expected 'divide by zero' to be found in file 'C:\Users\VssAdministrator\AppData\Local\Temp\6655aaa5-fed6-4160-ba8f-f1b718dc162d\4d6749bd-baa4-4f4e-bef9-b1f477bd0748', but it was not found.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 428
428:             $LogPath | Should -FileContentMatch 'divide by zero'
```